app/models/biocuration_classification.rb

```diff
@@ -24,7 +24,7 @@ class BiocurationClassification < ApplicationRecord
   acts_as_list scope: [:biological_collection_object_id, :project_id]
 
   belongs_to :biocuration_class, inverse_of: :biocuration_classifications
-  belongs_to :biological_collection_object, class_name: 'CollectionObject::BiologicalCollectionObject', inverse_of: :biocuration_classifications
+  belongs_to :biological_collection_object, class_name: '::CollectionObject::BiologicalCollectionObject', inverse_of: :biocuration_classifications
```

Comment:
Was `::` added by mistake or is actually required?

Reply:
Tried on console. Looks harmless.

Reply:
This is indeed necessary for namespacing some of the modules that have identical names.  This says "from the root, find the module", rather than "from the relative position of this <method>, find the module".

Reply:
I was fearing that for `type` columns it would attempt to match with leading `::` but I looks it is smart enough to not do that.

Reply:
Right.  It took me a while to figure it out when the problem came, but it's definitely tested/working pattern.